README.md

```diff
@@ -129,7 +130,7 @@ oneDNN supports systems meeting the following requirements:
 * Operating system with Intel 64 / Arm 64 / Power / IBMz architecture support
 * C++ compiler with C++11 standard support
 * [CMake] 2.8.12 or later
-* [Arm Compute Library (ACL)] for builds using ACL on AArch64.
+* [Arm Compute Library (ACL)] 24.04 or later for builds using ACL on AArch64.
```

Comment:
This section covers minimal dependencies. Please move ACL to [CPU Engine section](https://github.com/oneapi-src/oneDNN?tab=readme-ov-file#cpu-engine)

Reply:
There is already a section on ACL in [System Requirements](https://github.com/oneapi-src/oneDNN/blob/061ce4afec80d62cb040a4016a0ccb15e811eecb/README.md?plain=1#L102-L107) section, so we probably don't need to repeat this information. I can remove the L133 if you would like?

Reply:
Let's just move everything related to ACL into CPU Engine section.